vnext/Directory.Build.props

```diff
@@ -17,10 +17,10 @@
 
   <PropertyGroup Label="Configuration">
     <ProjectName Condition="'$(ProjectName)'==''">$(MSBuildProjectName)</ProjectName>
-
+    <ReactNativeWindowsBuildDir Condition="'$(ReactNativeWindowsBuildDir)'==''">$(ReactNativeWindowsDir)build</ReactNativeWindowsBuildDir>
```

Comment:
>build [](start = 104, length = 5)

usually this is called obj ?

Reply:
Yes, but from the beginning of vnext development, it was decided to use the more broadly adopted `build` for intermediate artifacts.
Similarly, we use `target` for the final artifacts`.